### PR TITLE
Scrape guide

### PR DESCRIPTION
**Pull request by thomasdebeus**

### CODE REVIEW

File: web-scraping/readme.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+* We want the title of each experience: 'Content developer'. ```class='item-title'```.
+* We also want the organisation of each title: 'New Atoms'. ```class='item-subtitle'```.
+* finally, we want the date and time the person was active under that title and organisation: ```class='.date-range'```
+
+This ↓ is all the code we need. Not very scary, aint it?
+
+    var x = new require('x-ray')()
+
+    x(
+      'https://linkedin.com/in/some-person',
```
Comment (bvhme) [inline web-scraping/readme.md]: I would keep this a real link, as it allows people to then copy and past the code.

**Comment by thomasdebeus** [comment]
Merge?

File: web-scraping/readme.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
+> **The goal of this guide:** You will learn how to extract information from certain web pages using [developer tools](guides/fixing-errors/readme.md), [node.js](http://blog.modulus.io/absolute-beginners-guide-to-nodejs) and [X-Ray](https://www.npmjs.com/package/x-ray).
+
```
Comment (SPrinss) [inline web-scraping/readme.md:2]: path to guides incorrect

File: web-scraping/readme.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+You can of course start here at [hackertyper](http://hackertyper.com/) and do your [swordfish](http://www.imdb.com/title/tt0244244/?ref_=nv_sr_1) magic! 🤓
+
+Web scraping enables you to extract information from the web and put it in a more workable format (*.json*, *.csv*, *excel*, *google spreadsheets*).     
+
+## First things first
+
+1. Create a file `'filename'.js` in your designated folder. In this file you will write your code.
+
+* Open the [Terminal](http://www.macworld.co.uk/feature/mac-software/get-more-out-of-os-x-terminal-3608274/) application on your mac, go to the folder where you created your files by typing `cd 'path to your folder'` enter.
+
```
Comment (SPrinss) [inline web-scraping/readme.md:14]: file, not files

File: web-scraping/readme.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+* Install node.js: `npm install node` and enter. Then you need X-Ray: `npm install x-ray` enter.
+
```
Comment (SPrinss) [inline web-scraping/readme.md:16]: not sure whether install node is necessary. Did you check this?

Comment (thomasdebeus) [inline web-scraping/readme.md:16]: Can't remember if I installed it I'll check it.

File: web-scraping/readme.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+In the Chrome web browser open the devtools with `cmd-alt-i` and click on the parent element:
+
+<img src="../images/devtools-web-scraping.png" width="800">
+
+> With scraping you need to get comfortable with developer tools and with reading html code.
+
+We search 🕵 from top to bottom to find what we want:
+
+* We want all the information of experience. `class='positions'`
+* We want the title of each experience: 'Content developer'. `class='item-title'`.
```
Comment (SPrinss) [inline web-scraping/readme.md]: 'Content developer'  not entirely clear that it is an example.

Perhaps add e.g.

File: web-scraping/readme.md [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+        title: '.item-title',
+        organisation: '.item-subtitle',        
+        time: '.date-range'
+      } ]
+    )
+  }
+).write('results.json')
+```
+
+1. Create a variable that will execute your x-ray. `var x = new require('x-ray')()`
```
Comment (SPrinss) [inline web-scraping/readme.md:56]: not sure whether it is execute. 

It imports a new instance of the module but doesn't run it. (yet)

File: web-scraping/readme.md [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+* Now `x` will look at a Linkedin url.
```
Comment (SPrinss) [inline web-scraping/readme.md:57]: so this is run execute the module with the following arguments

File: web-scraping/readme.md [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+* Next, the code will run the containing information of the called classes.
+* With `.write('results.json')` the scraped information will print out the information in a newly created `results.json` file on your computer.
+* To actually see 👀 your scraped information in your results.json file go to your terminal and call the code with typing: `node 'filename'.js`. Voila! There it is! 🎉
```
Comment (SPrinss) [inline web-scraping/readme.md:60]: will be printed out.
The information itself doesn't print

File: web-scraping/readme.md (hunk 7 above)
Comment (thomasdebeus) [inline web-scraping/readme.md:57]: yes

File: web-scraping/readme.md [hunk 9]
```diff
@@ -0,0 +1,146 @@
...
+1. Create a variable that will execute your x-ray. `var x = new require('x-ray')()`
+* Now `x` will look at a Linkedin url.
+* Next, the code will run the containing information of the called classes.
+* With `.write('results.json')` the scraped information will print out the information in a newly created `results.json` file on your computer.
+* To actually see 👀 your scraped information in your results.json file go to your terminal and call the code with typing: `node 'filename'.js`. Voila! There it is! 🎉      
+
+## Results in your .json file:
+
+You can convert these .json files into .csv files to eventually import it in spreadsheet programs or [Open Refine](http://openrefine.org/).
+
```
Comment (SPrinss) [inline web-scraping/readme.md:65]: How? Why?

**Comment by SPrinss** [comment]
I'm doubting whether it's a bug or not but I'd make sense to me to briefly explain the working of x-ray's input arguments. Since it's difficult to write your own script without knowledge of how it works.

For example: The [], means that it will take all instances with those parameters (instead of only the first).
The 
      '.position',
      [ {
        title: '.item-title',
means that it search for .item-title inside .position (if I'm correct)

**Comment by SPrinss** [comment]
Aside from that, I love it!

**Comment by thomasdebeus** [comment]
That's a bug. this draft is more a first start of this thing (better than what we had 😉). Also the more complex code which handles multiple urls and classes needs further explaining.

**Comment by SPrinss** [comment]
But if it's a bug than I can't merge it according to our rules :)

**Pull request merged**